index.bs

```diff
@@ -93,6 +96,8 @@ dictionary MediaStreamTrackProcessorInit {
 2. Let |p| be a new {{MediaStreamTrackProcessor}} object.
 3. Assign {{init.}}{{MediaStreamTrackProcessorInit/track}} to the {{MediaStreamTrackProcessor/[track]}}
    internal slot of |p|.
+3. Assign {{init.}}{{MediaStreamTrackProcessorInit/maxBufferSize}} to the {{MediaStreamTrackProcessor/[maxBufferSize]}}
```

Comment:
I note that init. is not a link, and there's an extra space in the formatting. Should {{init.}} be {{init}}. ?

Reply:
Nice catch!
Done